js-miniapp-bridge/src/index.ts

```diff
@@ -33,7 +33,7 @@ import {
   errorTypesDescriptions,
   MiniAppErrorType,
 } from './types/error-types';
-import { Product, PurchasedProduct } from './types/purchaseProduct';
```

Comment:
`Product`, `PurchasedProduct`, and `PurchasedProductResponseStatus` should also be exported from here and then exported from `js-mniniapp-sdk/src/index.ts`. Otherwise I think that the mini app can't properly import those types if they want to use them.

Reply:
Agreed. Pushed the changes @corycaywood